CMakeLists.txt

```diff
@@ -213,6 +212,18 @@ if(NOT WOLFSSL_ASM)
         "-DWOLFSSL_NO_ASM")
 endif()
 
+# Enable Debugging
+set(WOLFSSL_DEBUG_HELP_STRING "Enables option for debug (default: disabled)")
+add_option("WOLFSSL_DEBUG" ${WOLFSSL_DEBUG_HELP_STRING} "no" "yes;no")
+
+if(WOLFSSL_DEBUG)
+    list(APPEND WOLFSSL_DEFINITIONS
+        "-g"
```

Comment:
`-g` is a compiler flag, rather than a define/macro. Please add it to `CMAKE_C_FLAGS` instead. See [here](https://cmake.org/cmake/help/latest/variable/CMAKE_LANG_FLAGS.html).

Reply:
Fixed.